README.md

```diff
@@ -48,16 +48,19 @@ Paper: <br>[![][paper-img1]][paper-url1]
 | 📦 KomaMRI.jl        | [![][komamri-version]][komamri-juliahub]     | [![][gh-actions-img1]][gh-actions-url] | [![][codecov-img1]][codecov-url] |
 | └ 📦 KomaMRICore.jl  | [![][komacore-version]][komacore-juliahub]   | [![][gh-actions-img2]][gh-actions-url] | [![][codecov-img2]][codecov-url] |
 | └ 📦 KomaMRIPlots.jl | [![][komaplots-version]][komaplots-juliahub] | [![][gh-actions-img3]][gh-actions-url] | [![][codecov-img3]][codecov-url] |
+| └ 📦 KomaMRIIO.jl    | ![Static Badge](https://img.shields.io/badge/developing%20...%20-%20%23575757)    | [![][gh-actions-img4]][gh-actions-url] | [![][codecov-img4]][codecov-url] |
```

Comment:
It is better to point to something that do not exist yet than putting dev..

Reply:
Addressed in [f425357](https://github.com/cncastillo/KomaMRI.jl/commit/f425357a8ba66fd399901c8b0fe1dae2b08960e6)